tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -0,0 +1,269 @@
+Feature: check share activity
+  As a user
+  I want to check who shared which file to whom
+  So that I can track activities of a file
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
```

Comment:
@nirajacharya2  I think only the following two steps are suitable to be in the background:
``` feature
 Given these users have been created with default attributes and without skeleton files:
      | username |
      | Alice    |
      | Brian    |
 And user "Alice" has uploaded file with content "ownCloud test text file 0" to "textfile0.txt"
 ```
      
      
Because i am going to add scenarios from this PR: https://github.com/owncloud/ocis/pull/9803 to this feature file as well and they wont need the following step :

``` feature
 And user "Alice" has sent the following resource share invitation:
```